O365/sharepoint.py

```diff
@@ -236,8 +240,41 @@ def get_list_columns(self):
 
         return [self.list_column_constructor(parent=self, **{self._cloud_data_key: column})
                 for column in data.get('value', [])]
-
-
+    
+    def update_list_item(self, item_id, update):
+        """Updates a list item with a given update
+           
+        :param item_id: List item id to update
+        :param update: Data to update list item with {'col_name': 'Updated Data'}
+        
+        :rtype: SharepointListItem
+        """
+        
+        url = self.build_url(self._endpoints.get('get_item_by_id').format(item_id=item_id) + '/fields')
```

Comment:
This url should be added to endpoints.
The urls used must appear as endpoints.

Like this:

```python
_endpoints = {
    # ...
    'update_field': '/items/{item_id}/fields',
    # ...
}
```
Then this line should simply:

```python
url = self.build_url(self._endpoints.get('update_field').format(item_id=item_id))
```